.github/workflows/playwright.yml

```diff
@@ -5,9 +5,16 @@ jobs:
   test:
     timeout-minutes: 15
     runs-on: [self-hosted, asg]
-    # Certs added for the self hosted runner
-    env:
-      NODE_EXTRA_CA_CERTS: /etc/ssl/certs/ca-certificates.crt
+    container:
+      image: public.ecr.aws/docker/library/node:18.19.0-bullseye
```

Comment:
just for my clarity is this a public aws image or something from the va?

Reply:
These are all from AWS's public repository. We could create our own and store them in Private ECR repos too. Image pulls are pretty fast from here since I think they pull across AWS's network backbone and not the TIC.

Reply:
if we can pull from here & still install our own system deps (checks passed, looks like we can), i'm fine with this. we are still using the self hosted runner underneath it all